server/modules/survey/api/surveyApi.js

```diff
@@ -7,15 +7,20 @@ import * as JobUtils from '@server/job/jobUtils'
 import * as Validation from '@core/validation/validation'
 import * as User from '@core/user/user'
 
-import * as AuthMiddleware from '../../auth/authApiMiddleware'
-import * as SurveyService from '../service/surveyService'
-import * as UserService from '../../user/service/userService'
+import UnauthorizedError from '@server/utils/unauthorizedError'
+import * as AuthMiddleware from '@server/modules/auth/authApiMiddleware'
+import * as SurveyService from '@server/modules/survey/service/surveyService'
+import * as UserService from '@server/modules/user/service/userService'
 
 export const init = app => {
   // ==== CREATE
   app.post('/survey', async (req, res, next) => {
     try {
       const user = Request.getUser(req)
+      if (!User.isSystemAdmin(user)) {
+        throw new UnauthorizedError(User.getName(user))
+      }
```

Comment:
don't we have a requireAdminPermission middleware function?

Reply:
no but we can add it ;)

Reply:
cool